### PR TITLE
create Select User Login / SignUp Registration taps and design

### PR DESCRIPTION
**Pull request by shreyas1434shinde**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <router-outlet></router-outlet>
 <ng-template [ngIf]="isFooter">
-  <div class="pt-70"></div>
+  <!-- <div class="pt-70"></div> -->
```
Comment (itsvick) [inline src/app/app.component.html:3]: @shreyas1434shinde remove commented code

File: src/app/ekyc/ekyc.component.html [hunk 2]
```diff
@@ -17,7 +17,30 @@
...
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="d-flex justify-content-center">
+            <div>
+              <div class="fs-20 text-center text-grey2 font-600 ">
+                {{'LINK_YOUR_UDISE'| translate}}
+              </div>
+              <div>
+                <button _ngcontent-coo-c168="" [disabled]="aadharForm.invalid" type="submit" class="custom-button1 blue-button mt-20 pr-90 fs-14">{{'LINK_NOW'| translate}} </button>
```
Comment (itsvick) [inline src/app/ekyc/ekyc.component.html:38]: @shreyas1434shinde what is "_ngcontent-coo-c168" here, remove if not required

File: src/app/ekyc/ekyc.component.html [hunk 3]
```diff
@@ -17,7 +17,30 @@
       <p class="text-danger" *ngIf="aadharId.invalid && aadharId.touched">{{'INCORRECT_AADHAR' | translate}}</p>
     </div>
     <div class="pb-10">
-      <button type="submit" class="btn btn-primary mt-4" [disabled]="aadharForm.invalid">{{'VERIFY'| translate}}</button>
+      <button type="submit" class="btn btn-primary mt-4"  data-toggle="modal" data-target="#exampleModalCenter">{{'VERIFY'| translate}}</button>
```
Comment (itsvick) [inline src/app/ekyc/ekyc.component.html:20]: @shreyas1434shinde can u keep aadhar form validation here.
[disabled]="aadharForm.invalid"

File: src/app/login-manav-sampada/login-manav-sampada.component.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      <hr class="horizontal-line" />
+ 
+      <div class="text-center mt-5 pt-5 d-flex jc-content-center">
+        <div class="square pt-4"><img src="assets/images/beo.png" alt="">
+            <div class="py-2 pt-3 fs-16 font-600">{{'BLOCK_EDUCATIONAL_OFFICER' | translate }}</div>
+        </div>
+      </div>
+    </div>
+    <div class="d-flex justify-content-center mt-4">
+        <button (click)="openSSO('login-auth-BEO')" type="button" class="custom-button blue-button mt-20 pr-90" >Login using Meri Pehchaan</button>
```
Comment (itsvick) [inline src/app/login-manav-sampada/login-manav-sampada.component.html:30]: @shreyas1434shinde Add translation for `Login using Meri Pehchaan`

File: src/app/login-manav-sampada/login-manav-sampada.component.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+import { environment } from 'src/environments/environment';
+
+@Component({
+  selector: 'app-login-manav-sampada',
+  templateUrl: './login-manav-sampada.component.html',
+  styleUrls: ['./login-manav-sampada.component.scss']
+})
+export class LoginManavSampadaComponent implements OnInit {
+  baseUrl: string;
+  featureList = [
```
Comment (itsvick) [inline src/app/login-manav-sampada/login-manav-sampada.component.ts:18]: @shreyas1434shinde This list should be translated as well.

File: src/app/meri-pehchaan/meri-pehchaan.component.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      <hr class="horizontal-line" />
+ 
+      <div class="text-center mt-5 pt-5 d-flex jc-content-center">
+        <div class="square pt-4"><img src="assets/images/profile.png" alt="">
+          <div class="py-2 pt-3 fs-16 font-600"> {{'PRINCIPAL' | translate }}</div>
+        </div>
+      </div>
+    </div>
+    <div class="d-flex justify-content-center mt-4">
+        <button (click)="openSSO('login-auth-principal')" type="button" class="custom-button blue-button mt-20 pr-90 fs-16" >Login using Meri Pehchaan</button>
```
Comment (itsvick) [inline src/app/meri-pehchaan/meri-pehchaan.component.html:30]: @shreyas1434shinde translate this as well

File: src/app/registration/registration.component.html [hunk 7]
```diff
@@ -1,40 +1,141 @@
...
+              
+             
+            </ul>
+            <!-- end design process steps--> 
+            <!-- Tab panes -->
+            <div class="tab-content mt-5">
+              <div role="tabpanel" class="tab-pane active" id="discover">
+                <div class="fs-24 text-white font-600">Please loginto AVSAR with your Meri-Pehchaan credentials</div>
+                <div class="fs-18 pt-4 text-white font-400">
+                  1. If you have digilocker login with Pin enter your phone number on mobile tab and pin or username and pin on username tab or aadhaar and pin in others tab.
```
Comment (itsvick) [inline src/app/registration/registration.component.html:99]: @shreyas1434shinde add translations

File: src/app/registration/registration.component.html [hunk 8]
```diff
@@ -1,40 +1,141 @@
...
+                <div class="mt-5 mb-5">
+                  <div class="iframe-container1">
+                    <iframe class="responsive-iframe" width="582px" height="276px" src="https://www.youtube.com/embed/D0UnqGm_miA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                  </div>
+                </div>
+              </div>
+              <div role="tabpanel" class="tab-pane" id="optimization">
+                <div class="mt-5 mb-5">
+                  <div class="iframe-container1">
+                    <iframe class="responsive-iframe" width="582px" height="276px" src="https://www.youtube.com/embed/D0UnqGm_miA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (itsvick) [inline src/app/registration/registration.component.html:120]: @shreyas1434shinde we need to make this url configurable.

File: src/app/registration/registration.component.ts [hunk 9]
```diff
@@ -7,6 +7,7 @@ import { IImpressionEventInput, IInteractEventInput } from '../services/telemetr
 import { TelemetryService } from '../services/telemetry/telemetry.service';
 import { ToastMessageService } from '../services/toast-message/toast-message.service';
 import { environment } from 'src/environments/environment';
+declare var $: any;
```
Comment (itsvick) [inline src/app/registration/registration.component.ts:10]: @shreyas1434shinde Avoid using jQuery

File: src/app/app.component.html (hunk 1 above)
Comment (shreyas1434shinde) [reply src/app/app.component.html:3]: @itsvick done

File: src/app/ekyc/ekyc.component.html (hunk 3 above)
Comment (shreyas1434shinde) [reply src/app/ekyc/ekyc.component.html:20]: @itsvick done

File: src/app/login-manav-sampada/login-manav-sampada.component.html (hunk 4 above)
Comment (shreyas1434shinde) [reply src/app/login-manav-sampada/login-manav-sampada.component.html:30]: @itsvick done

File: src/app/meri-pehchaan/meri-pehchaan.component.html (hunk 6 above)
Comment (shreyas1434shinde) [reply src/app/meri-pehchaan/meri-pehchaan.component.html:30]: @itsvick done

File: src/app/registration/registration.component.html (hunk 8 above)
Comment (shreyas1434shinde) [reply src/app/registration/registration.component.html:120]: @itsvick yes it's youtube iframe it is configurable

File: src/assets/i18n/en.json [hunk 10]
```diff
@@ -264,10 +264,17 @@
...
-  "PRINCIPAL": "Principal",
-  "BLOCK_EDUCATIONAL_OFFICER": "Block Educational Officer",
+  "PRINCIPAL": "School Principal",
+  "BLOCK_EDUCATIONAL_OFFICER": "BEO",
+  "Select_User":"Select User",
+  "Login_SignUp":"Login / SignUp",
+  "Registration_Form":"Registration Form",
+  "Please_loginto":"Please loginto AVSAR with your Meri-Pehchaan credentials",
+  "If_you_have_digilocker":"1. If you have digilocker login with Pin enter your phone number on mobile tab and pin or username and pin on username tab or aadhaar and pin in others tab.",
+  "If_you_dont_remember":" 2. If you dont remember your digilocker pin, then download digilocker and create account and set pin and login as per instruction 1.",
```
Comment (itsvick) [inline src/assets/i18n/en.json:274]: @shreyas1434shinde keys should be in BLOCK letters

**Pull request merged**